jsonnet/prometheus.jsonnet

```diff
@@ -156,6 +156,9 @@ local namespacesRole =
       secret.mixin.metadata.withNamespace($._config.namespace) +
       secret.mixin.metadata.withLabels({ 'k8s-app': 'prometheus-k8s' }),
 
+
+
+
```

Comment:
out of curiosity: are those three newlines on purpose?

Reply:
Done, thanks!